sdcm/tester.py

```diff
@@ -3172,18 +3171,29 @@ def fixture_event_system(self, setup_logging):
         yield
         self.stop_event_device()
 
+    @pytest.fixture(autouse=True, name='threads_checks', scope='session')
+    def fixture_threads_checks(self):
+
+        yield
+
+        self._check_alive_routines_and_report_them()
+        self._check_if_db_log_time_consistency_looks_good()
+        logging.debug("Threads and processes at the end of the test:")
+        for t in threading.enumerate():
+            logging.debug(f"Active thread: {t.name} (id={t.ident}, daemon={t.daemon}, repr={repr(t)})")
+
     @silence()
     def _check_if_db_log_time_consistency_looks_good(self):
```

Comment:
It's been some time since we had issues with log time consistency. maybe not needed anymore?